src/catalog.c

```diff
@@ -629,15 +629,23 @@ ts_catalog_restore_user(CatalogSecurityContext *sec_ctx)
 /*
  * Insert a new row into a catalog table.
  */
-void
-ts_catalog_insert(Relation rel, HeapTuple tuple)
+TSDLLEXPORT void
+ts_catalog_insert_only(Relation rel, HeapTuple tuple)
```

Comment:
seems like you didnt add any external callers for this can you explain when you would use this

Reply:
It's called from the new invalidation code in `tsl/src/continuous_aggs/invalidation.c`:
https://github.com/timescale/timescaledb/blob/5dd3cb144b792e56b66392c39c0cdaaaa0ac9daa/tsl/src/continuous_aggs/invalidation.c#L331

Reply:
I guess I can remove the export here though, since it is in the header file.